domain/BUILD.bazel

```diff
@@ -10,7 +10,10 @@ load("//domain:domain_test.bzl", "domain_test")
 
 kt_android_library(
     name = "domain",
-    srcs = glob(["src/main/java/org/oppia/android/domain/**/*.kt"]),
+    srcs = glob(
+        ["src/main/java/org/oppia/android/domain/**/*.kt"],
```

Comment:
Good question--it's definitely better to be specific. The reason we aren't here is because we explicitly need domain to include everything (in case new files are added). Phase 2 of the Bazel migration involves splitting this up into smaller, directory-based modules. We wouldn't use globs for those which means long-term we wouldn't use wildcards anymore.